3.10/release-notes-new-features310.md

```diff
@@ -9,6 +9,13 @@ The following list shows in detail which features have been added or improved in
 ArangoDB 3.10. ArangoDB 3.10 also contains several bug fixes that are not listed
 here.
 
+Native ARM Support
+------------------
+
+ArangoDB is now available for the ARM architecture, in addition to the x86-64
+architecture. It can run on Apple silicon (e.g. M1 chips) without emulation.
+It also runs on other ARM chips under Linux and Windows for ARM.
```

Comment:
Will we support Windows for ARM? Or only Linux and macOS?
What is the minimum ARM architecture that we will support? ARMv8 (because of SIMD/NEON)?

Reply:
@Simran-B AFAIK no Windows support is planned. Yes, it will be ARMv8 at this point and NEON is used.